### PR TITLE
Add util tests

### PR DESCRIPTION
**Pull request by flrnd**
Related to #23 
I've added few tests. One of them need some tests cases, I'm slow today and can't figure out what should be the correct inputs.

### CODE REVIEW

File: utils/utils_test.go [hunk 1]
```diff
@@ -53,3 +53,41 @@ func TestGetGithubBaseURL(t *testing.T) {
...
+	}{
+		{"foo.go", true},
+		{"bar.gol", false},
+		{"foobar", false},
+		{"fubar.g", false},
+	}
+	for _, tt := range tests {
+		t.Run(fmt.Sprintf("given the filename %s", tt.got), func(t *testing.T) {
+			got := IsGoFile(tt.got)
+			assert.Equal(t, got, tt.want)
```
Comment (rodrigo-brito) [inline utils/utils_test.go]: The assertion is inverted
```suggestion
			assert.Equal(t, tt.want, got)
```

File: utils/utils_test.go [hunk 2]
```diff
@@ -53,3 +53,41 @@ func TestGetGithubBaseURL(t *testing.T) {
...
+	}{
+		// Need some test cases, but not sure what this function really does.
+	}
+	for _, tt := range tests {
+		t.Run(fmt.Sprintf("Trim GOPATH"), func(t *testing.T) {
+			got := TrimGoPath(tt.args.path, tt.args.repository)
+			assert.Equal(t, got, tt.want)
+		})
+	}
+}
```
Comment (rodrigo-brito) [inline utils/utils_test.go]: This is a good test dor this function.
```suggestion
func TestTrimGoPath(t *testing.T) {
	tests := []struct {
		path string
		repository string
		want string
	}{
		{fmt.Sprintf("%s/src/gocity/main.go", os.Getenv("GOPATH")), "gocity", "/main.go"},
		{fmt.Sprintf("%s/src/gocity/foo/bar.go", os.Getenv("GOPATH")), "gocity", "/foo/bar.go"},
		{fmt.Sprintf("%s/src/gocity/vendor", os.Getenv("GOPATH")), "gocity", "/vendor"},
	}
	for _, tt := range tests {
		t.Run(fmt.Sprintf("given project %s/%s", tt.path, tt.repository), func(t *testing.T) {
			got := TrimGoPath(tt.path, tt.repository)
			assert.Equal(t, tt.want,  got)
		})
	}
}
```

Comment (flrnd) [reply utils/utils_test.go]: Hi Rodrigo! Thanks! I see now, silly me 😅

**Comment by rodrigo-brito** [comment]
This is great! Thank you @flrnd!

**Comment by flrnd** [comment]
@rodrigo-brito My pleasure!

**Pull request merged**